setup.cfg

```diff
@@ -9,6 +9,10 @@ long_description_content_type = text/markdown
 url = https://github.com/zhinst/zhinst-toolkit
 project_urls =
     Bug Tracker = https://github.com/zhinst/zhinst-toolkit/issues
+    Documentation = https://docs.zhinst.com/zhinst-toolkit/en/latest/
+    Release notes = https://docs.zhinst.com/zhinst-toolkit/en/latest/changelog/index.html
```

Comment:
I would point the release notes to the GitHub original source. This would also show unreleased changes so it is easier to see what will come with the next version. What do you think?

Reply:
I would keep current releases and incoming features separate.